source/code-generation/query-converter/bulk-conversion.txt

```diff
@@ -0,0 +1,100 @@
+.. _rm-bulk-conversion:
+
+===============
+Bulk Conversion
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can select and convert one or more queries, stored procedures, 
+triggers, and views in bulk by enabling the multi-select toggle. 
+Bulk conversion creates a job that queues and executes each query 
+conversion individually from a single request.
+
+Copy and paste your queries into the query converter individually and 
+then bulk convert a selection to update them to work with MongoDB and
+your migrated schema.
```

Comment:
I would drop this; it's not specifically about bulk conversions, and it's only describing one way of getting queries into the system.